### ISSUE TITLE
Landing Page Design Rework

### ISSUE THREAD

**Issue by 007jithinjose** [Morphclue/apollusia#192]
so i have been making a landing page design
[](https://www.figma.com/design/RcB4OmiSruBhKwidKA9ZYm/Apollusia?node-id=17-20&t=8DNQI8mB9x2lNqvW-1)
Can you look into and see if you are ok with it.
if everything is ok i can work on it

**Comment by Morphclue** [comment]
First of all thanks for taking the time to improve our first time visitor experience. Here are my thoughts about all segments. @Clashsoft I would like you to expand on this and give your own 2cents. I try to give my feedback from the perspective of a first time user. I'll make numbered lists for easier reference here in the discussion.

### General
1. I think instead of Real-Time usage we should advertise the `100% free to use` feature. e.g. like it's done in the [About](https://apollusia.com/about) page. 
2. Was darkmode considered?

### Navbar
1. A lot of the information that we currently have in our navbar has been lost. There are no recent polls, changelog and so on. I would suggest that we keep the current navbar, as we have way more information that could be visually displayed in a simple navbar without any burger menus. 
4. The thing that this navbar does better is highlighting the option to sign in for the first time users. We need to re-consider if we highlight this button when the user isn't signed in. I'll leave this up for discussion.

### First info Segment
(Headline, Subtext and Create a Poll-Button)
It shows clear intent and summarizes the basic fuctionalities in a headline. Furthermore there is some additional information that highlights the efficiency of this tool. I think it looks clear for users. 

### Example Screenshot with Text
Always a good idea to showcase some usage from the application!
1. The text right next to it needs to be worked upon and I'm not sure if we should use muted /greyed out text there, since it looks a bit off in my opinion.

### Cards
1. Cards look a bit uninteresting, because they are completly greyed out. Is there some idea behind it that we will do something with them on:hover? 
2. Maybe add some icons?

### Coperate Icons 
(I think that's how they are called, but I'm not sure)
1. The overall style looks good, but it just repeats the information we were given several times before. 
2. Do we have a license for these icons? Are they generated?

### Footer
1. Generally a good idea to have one, but we don't have any social accounts that we could link to. It could make sense to add legal info etc. there, but on the other side that info is already given in the navbar.

**Comment by 007jithinjose** [comment]
https://www.figma.com/proto/RcB4OmiSruBhKwidKA9ZYm/Apollusia?node-id=15-3&node-type=frame&t=dZ2dkNgMpUw1xcH3-1&scaling=min-zoom&content-scaling=fixed&page-id=0%3A1

So made some changes, and moved the features in the sidebar to the footer. if this is ok ill start dev

**Comment by Morphclue** [comment]
> So made some changes, and moved the features in the sidebar to the footer

The burgermenu / navbar shouldn't be changed for now. I think we can improve this in a subsequent PR / Issue if necessary. Moving the items to the footer has also lost the information for all "Recent Polls". I think it would also make more sense to refactor it later, if needed at all, since I currently see no issues with the current navbar except the possible improvement of the Sign in button being made more clear. But we will need to validate that with some users first. 

> if this is ok I'll start dev.

It would be great if you could wait for @Clashsoft reply, since he is the other maintainer that needs to approve the changes. Afterwards we need to clarify all points first, before we start the development to avoid unwanted design changes.

> if you want I can use some other icons , that you suggest

We currently use bootstrap-icons in our project (see [package.json](https://github.com/Morphclue/apollusia/blob/master/package.json)).

**Comment by 007jithinjose** [comment]
ok so what if i just kept the sidebar too? would that make it ok since i wont miss anything else either

**Comment by Clashsoft** [comment]
Yes, you should keep the sidebar. I don't mind adding stuff to the navbar either as long as it works in mobile.
Regarding the design:
- The copy (text blocks) is sometimes a little wonky or even wrong. Ex: "Customize your poll with different question types", "Choose from various question formats like multiple choice, ratings to match your needs". Apollusia is not a generic poll platform, it's just for finding time slots. Maybe we should just highlight a few of the features in the Readme/About.
- "How it works" is almost unreadable
Other than that it seems fine. If you start working on it, I suggest you create a new page like `home` and place it under the `/` route, so you don't have to change the dashboard (which you call My Polls)

**Comment by 007jithinjose** [comment]
Ok I'll do that

**Comment by Clashsoft** [comment]
Screenshotting here in case the designs get lost:

<img width="1442" alt="Image" src="https://github.com/user-attachments/assets/0e9723ea-57da-4d59-8cd4-5fcf5f9c498c" />

<img width="1442" alt="Image" src="https://github.com/user-attachments/assets/f2eb9f85-a979-43c0-882b-4e1ab131fad9" />

<img width="1442" alt="Image" src="https://github.com/user-attachments/assets/4a47f359-e973-4d4b-b3e8-a47b8b6cd489" />

<img width="1442" alt="Image" src="https://github.com/user-attachments/assets/1013d5a7-9c84-4745-8edb-0708f7c20075" />

<img width="1442" alt="Image" src="https://github.com/user-attachments/assets/459230b1-85f6-4c56-a5b8-675847c1a845" />

**Comment by Clashsoft** [comment]
@Morphclue should we close this as Not Planned?

**Comment by Morphclue** [comment]
@Clashsoft I think we can leave it open for now, as we should still implement more information for SEO etc.
